public/app/plugins/datasource/tempo/traceql/TempoQueryBuilderOptions.tsx

```diff
@@ -31,162 +32,173 @@ const parseIntWithFallback = (val: string, fallback: number) => {
-    `Spans Limit: ${query.spss || DEFAULT_SPSS}`,
-    `Table Format: ${query.tableType === SearchTableType.Traces ? 'Traces' : 'Spans'}`,
-    '|',
-    `Streaming: ${searchStreaming ? 'Enabled' : 'Disabled'}`,
-  ];
-
-  const collapsedMetricsOptions = [
-    `Step: ${query.step || 'auto'}`,
-    `Type: ${query.metricsQueryType === MetricsQueryType.Range ? 'Range' : 'Instant'}`,
-    '|',
-    `Streaming: ${metricsStreaming ? 'Enabled' : 'Disabled'}`,
-    // `Exemplars: ${query.exemplars !== undefined ? query.exemplars : 'auto'}`,
-  ];
-
-  return (
-    <EditorRow>
-      <div className={styles.options}>
-        <QueryOptionGroup
-          title="Search Options"
-          collapsedInfo={collapsedSearchOptions}
-          isOpen={isOpen}
-          onToggle={toggleOpen}
-        >
-          <EditorField label="Limit" tooltip="Maximum number of traces to return.">
-            <AutoSizeInput
-              className="width-4"
-              placeholder="auto"
-              type="number"
-              min={1}
-              defaultValue={query.limit || DEFAULT_LIMIT}
-              onCommitChange={onLimitChange}
-              value={query.limit}
-            />
-          </EditorField>
-          <EditorField label="Span Limit" tooltip="Maximum number of spans to return for each span set.">
-            <AutoSizeInput
-              className="width-4"
-              placeholder="auto"
-              type="number"
-              min={1}
-              defaultValue={query.spss || DEFAULT_SPSS}
-              onCommitChange={onSpssChange}
-              value={query.spss}
-            />
-          </EditorField>
-          <EditorField label="Table Format" tooltip="How the query data should be displayed in the results table">
-            <RadioButtonGroup
-              options={[
-                { label: 'Traces', value: SearchTableType.Traces },
-                { label: 'Spans', value: SearchTableType.Spans },
-              ]}
-              value={query.tableType}
-              onChange={onTableTypeChange}
-            />
-          </EditorField>
-          <EditorField label="Streaming" tooltip={<StreamingTooltip />} tooltipInteractive>
-            <div>{searchStreaming ? 'Enabled' : 'Disabled'}</div>
-          </EditorField>
-        </QueryOptionGroup>
-
-        <QueryOptionGroup
-          title="Metrics Options"
-          collapsedInfo={collapsedMetricsOptions}
-          isOpen={isOpen}
-          onToggle={toggleOpen}
-        >
-          <EditorField
-            label="Step"
-            tooltip="Defines the step for metric queries. Use duration notation, for example 30s or 1m"
+export const TempoQueryBuilderOptions = React.memo<Props>(
+  ({ onChange, query, searchStreaming, metricsStreaming, app }) => {
+    const styles = useStyles2(getStyles);
+    const [isOpen, toggleOpen] = useToggle(false);
+    const isAlerting = app === CoreApp.UnifiedAlerting || app === CoreApp.CloudAlerting;
```

Comment:
Wherever this const is used are the changed I made in this file. The pre-commit hook decided to reformat parts of the file!

Reply:
AFAIK We don't want to support `CoreApp.CloudAlerting`. It's a legacy thing.
just `app === CoreApp.UnifiedAlerting` should be enough